helm/rook-operator/charts/rook-ceph-cluster/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+description: Manages a single Ceph cluster namespace for Rook
+name: rook-ceph-cluster
+version: 0.0.1
```

Comment:
is this version correct?

Reply:
yeah, this is just the version set in the upstream chart - i plan on sticking with it unless we need to change anything in this subchart